store/displayPreferences.ts

```diff
@@ -134,170 +98,116 @@ export const actions: ActionTree<
         }
       );
 
-      if (response.status !== 200)
+      if (response.status !== 200) {
         throw new Error(
           'get display preferences status response = ' + response.status
         );
+      }
 
-      await dispatch('initStateSuccess', { displayPreferences: response.data });
+      const data = castResponse(response.data);
+      if (!data.CustomPrefs) {
+        Object.assign(state, data.CustomPrefs);
```

Comment:
What does this mean is JS? When the data.customPrefs is empty, we assign it to our state?

Reply:
Thanks, this one I was able to figure it out :p But why assign if `!data.customPrefs` is true?